pytorch_pfn_extras/training/extension.py

```diff
@@ -126,8 +126,44 @@ def load_state_dict(self, to_load):
         pass
 
 
-def make_extension(trigger=None, default_name=None, priority=None,
-                   finalizer=None, initializer=None, on_error=None):
+class _CustomExtension(Extension):
```

Comment:
Better name it `_CallableExtension` to align with `_CallableTrigger` proposed in https://github.com/pfnet/pytorch-pfn-extras/pull/179 ?